hsmodels/schemas/fields.py

```diff
@@ -25,35 +25,41 @@ class Config:
         description="String expressing the Full text citation, URL link for, or description of the related resource",
     )
 
-    _parse_relation = root_validator(pre=True)(parse_relation)
+    _parse_relation = model_validator(mode='before')(parse_relation)
 
 
 class CellInformation(BaseMetadata):
     """
     A class used to represent the metadata associated with raster grid cells in geographic raster aggregations
     """
 
-    class Config:
-        title = 'Raster Cell Metadata'
+    model_config = ConfigDict(title='Raster Cell Metadata')
 
     # TODO: Is there such a thing as "name" for CellInformation?
-    name: str = Field(default=None, max_length=500, title="Name", description="Name of the cell information")
-    rows: int = Field(default=None, title="Rows", description="The integer number of rows in the raster dataset")
+    name: str = Field(default=None, max_length=500, title="Name", description="Name of the cell information",
+                      json_schema_extra={"rdf_predicate": HSTERMS.name})
+    rows: int = Field(default=None, title="Rows",
+                      description="The integer number of rows in the raster dataset",
+                      json_schema_extra={"rdf_predicate": HSTERMS.rows})
     columns: int = Field(
-        default=None, title="Columns", description="The integer number of columns in the raster dataset"
+        default=None, title="Columns", description="The integer number of columns in the raster dataset",
+        json_schema_extra={"rdf_predicate": HSTERMS.columns}
     )
     cell_size_x_value: float = Field(
         default=None,
         title="Cell size x value",
         description="The size of the raster grid cell in the x-direction expressed as a float",
+        json_schema_extra={"rdf_predicate": HSTERMS.cellSizeXValue}
     )
     cell_data_type: str = Field(
-        default=None, max_length=50, title="Cell data type", description="The data type of the raster grid cell values"
+        default=None, max_length=50, title="Cell data type", description="The data type of the raster grid cell values",
+        json_schema_extra={"rdf_predicate": HSTERMS.cellDataType}
```

Comment:
These HSTERMS were implied before (using the field attribute name)?  This means the term for this one is changing, so is this a correction in the term name?  What happened to the implied term naming?  I see the term is declared for `name`, `rows`, and `columns` as well.  But I'm not seeing the term declared for `Creator.name` below.

Reply:
I moved the `rdf_predicate` settings to the corresponding RDF models where we originally had it as part of the model Config class.